src/server/src/main/java/io/cassandrareaper/core/RepairUnit.java

```diff
@@ -115,5 +116,42 @@ private Builder(RepairUnit original) {
     public RepairUnit build(UUID id) {
       return new RepairUnit(this, id);
     }
+
+    @Override
+    public int hashCode() {
```

Comment:
Need a comment here about why 7 and 59 are significant. It may even be worth assigning them to a `static final int` variable.